mypy.ini

```
@@ -0,0 +1,12 @@
+[mypy]
+files = docs,share,src,tests
+ignore_missing_imports = true
```

This comment was lost:
```suggestion
ignore_missing_imports = true  # FIXME: get rid of missing imports ignore
```

---

I'll add it back.